components/Storyblok/SbGrid.tsx

```diff
@@ -1,11 +1,16 @@
 import { storyblokEditable, type SbBlokData } from '@storyblok/react/rsc';
 import { Grid, type GridGapType, type GridNumColsType } from '../Grid';
+import { WidthBox, type WidthType } from '../WidthBox';
 import { CreateBloks } from '../CreateBloks';
 
 type SbGridProps = {
   blok: {
     _uid: string;
     gap?: GridGapType;
+    boundingWidth?: 'site' | 'full';
+    width?: WidthType;
```

Comment:
Slight tweak to Grid options to have `boundingWidth` and `Width`

boundingWidth: the width of the bounding box that is the outer container, either edge-to-edge or centered-container

width: within the bounding box, number of columns that the grid takes up. Eg, it can be 6 of 12 columns of a centered container, or 8 of 12 columns of an edge-to-edge container.